src/main/fc/rc_adjustments.c

```diff
@@ -144,7 +144,7 @@ static const adjustmentConfig_t defaultAdjustmentConfigs[ADJUSTMENT_FUNCTION_COU
     }, {
         .adjustmentFunction = ADJUSTMENT_RATE_PROFILE,
         .mode = ADJUSTMENT_MODE_SELECT,
-        .data = { .switchPositions = 3 }
+        .data = { .switchPositions = CONTROL_RATE_PROFILE_COUNT }
```

Comment:
We should not do this. Expecting everybody to buy a new TX that comes with a 6 position switch, or modify the hardware they already own is just not realistic.

Reply:
I don't expect many people to have a six position switch. However on Taranis and DeviationTx it is easy to multiplex together a 3-position switch and a 2-position switch to give a 6-position switch.

And if users don't want to do this, they can just adjust the endpoints of the switch so it activates the first 3 values.

Reply:
This seems to be making an established functionality more clumsy and harder to configure. As a suggestion, how about we add `ADJUSTMENT_RATE_PROFILE_6POS` as an additional adjustment mode, and leave `ADJUSTMENT_RATE_PROFILE` the way it is?

Reply:
I like the idea of adding a separate 6POS adjustment mode and leaving the existing 3 position mode intact.  Otherwise you'll break the setup for everybody who is already using a 3 position adjustment mode.  The few that want to use more than 3 should be the ones who need to configure something new.